More info to get started with Flutter (#1069)
WalletConnect/walletconnect-docs: docs/web3modal/flutter/installation.mdx

## Installation

> Check on [pu.dev](https://pub.dev/packages/web3modal_flutter/versions) the version you would like to install and add it as dependency in your pubspec.yaml (and run an implicit flutter pub get):

Reviewer: ```suggestion
Check out [pub.dev](https://pub.dev/packages/web3modal_flutter/versions) for the version you would like to install and add it as dependency in your pubspec.yaml (and run an implicit `flutter pub get` ):
```

```ruby
dependencies:
  web3modal_flutter: ^3.0.0-beta01
```

### Enable Installed Wallet Detection

To enable Web3Modal to detect wallets installed on the device, you need to make specific changes to the native sides of the project.

> #### For iOS:

Reviewer: Seems good for now, but in future we could use Platform Tabs instead for this (check main README)

Author: Done! Check it out!

Reviewer: Amazing! Thanks for the changes, LGTM
